File: safari-600.7-branch/LayoutTests/http/tests/security/resources/abe-allow-credentials.php
<?php
header("Access-Control-Allow-Origin: http://127.0.0.1:8000");
header("Access-Control-Allow-Credentials: true");

$name = 'abe.png';
$fp = fopen($name, 'rb');
header("Content-Type: image/png");
header("Content-Length: " . filesize($name));

fpassthru($fp);
exit;
?>

File: LayoutTests/http/tests/security/resources/abe-allow-credentials.py
#!/usr/bin/env python3

import os
import sys

sys.stdout.write(
    'Access-Control-Allow-Origin: http://127.0.0.1:8000\r\n'
    'Access-Control-Allow-Credentials: true\r\n'
    'Content-Type: image/png\r\n\r\n'
)

sys.stdout.flush()
with open(os.path.join(os.path.dirname(__file__), 'abe.png'), 'rb') as file:
    sys.stdout.buffer.write(file.read())
